hkparser.js

```diff
@@ -4,26 +4,27 @@
  */
 "use strict";
 
-const HKUris 			= require("./hk");
-const Utils 			= require("./utils");
-const Constants			= require("./constants");
-const xml               = require("./xmlschema");
+const HKUris = require("./hk");
+const Utils = require("./utils");
+const Constants = require("./constants");
+const xml = require("./xmlschema");
 
-const HK				= require("hklib");
+const HK = require("hklib");
 
-const Node              = HK.Node;
-const Trail             = HK.Trail;
-const Connector         = HK.Connector;
-const Link              = HK.Link;
-const Context           = HK.Context;
-const Reference         = HK.Reference;
-const LAMBDA            = HK.Constants.LAMBDA;
+const Node = HK.Node;
+const Trail = HK.Trail;
+const Connector = HK.Connector;
+const Link = HK.Link;
+const Context = HK.Context;
+const Reference = HK.Reference;
+const LAMBDA = HK.Constants.LAMBDA;
 
 const HYPERKNOWLEDGE_URIS = new Set();
 
 HYPERKNOWLEDGE_URIS.add(HKUris.HAS_PARENT_URI);
 HYPERKNOWLEDGE_URIS.add(HKUris.REFERENCES_URI);
 HYPERKNOWLEDGE_URIS.add(HKUris.USES_CONNECTOR_URI);
+HYPERKNOWLEDGE_URIS.add(HKUris.REFERENCED_BY_URI);
```

Comment:
REFERENCED_BY_URI  has been previously deprecated. Is the any specific need for adding it back to the lib?

Reply:
Not really, I must have worked on an earlier commit. I'll remove it.